bigtable/admin.go

```diff
@@ -1584,14 +1584,21 @@ func UpdateInstanceAndSyncClusters(ctx context.Context, iac *InstanceAdminClient
 
 // RestoreTable creates a table from a backup. The table will be created in the same cluster as the backup.
 func (ac *AdminClient) RestoreTable(ctx context.Context, table, cluster, backup string) error {
-	ctx = mergeOutgoingMetadata(ctx, ac.md)
-	prefix := ac.instancePrefix()
-	backupPath := ac.backupPath(cluster, backup)
+	return ac.RestoreTableFrom(ctx, ac.instance, cluster, table, backup)
+}
 
+// RestoreTableFrom creates a new table in the admin's instance by restoring from the specified backup in a different instance.
+// sourceInstance (ex. "my-instance") and sourceCluster (ex. "my-cluster") are the instance and cluster in which the new table will be restored from.
+// tableName (ex. "my-restored-table") will be the name of the newly created table
+// backupName (ex. "my-backup") is the name of the backup to restore
+func (ac *AdminClient) RestoreTableFrom(ctx context.Context, sourceInstance, sourceCluster, tableName, backupName string) error {
```

Comment:
Docs are much clearer, thanks!

Nit: could you switch the args to `table` and `backup` to match `RestoreTable` above? It looks weird that the names are different.